packages/api/endpoints/workflows.js

```diff
@@ -26,20 +30,28 @@ function list(event, cb) {
  * @param {callback} cb - aws lambda callback function
  * @returns {undefined} undefined
  */
-function get(event, cb) {
+async function get(event, cb) {
   const name = _get(event.pathParameters, 'name');
```

Comment:
You don't need to use `_get()` here.  The conditional on line 66 is ensuring that `pathParameters` will be set

Reply:
Changed